help/destinations/catalog/mobile-engagement/moengage.md

```diff
@@ -0,0 +1,141 @@
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+See [Activate audience data to streaming segment export destinations](../../ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from [!DNL Adobe Experience Platform] to the [!DNL Moengage] destination, you need to go through the field mapping step.
+
+Mapping consists of creating a link between your [!DNL Experience Data Model] (XDM) schema fields in your [!DNL Platform] account, and their corresponding equivalents from the target destination.
+
+To correctly map your XDM fields to the [!DNL Braze] destination fields, follow these steps:
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Checkbox]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/segments.png)
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/mapping.png)
+
+In the [!UICONTROL Source Field] section, click the arrow button next to the empty field.
+   
+![Moengage Destination Source Mapping](../../assets/catalog/mobile-engagement/moengage/mapping-source.png)
+
+In the [!UICONTROL Select source field] window, you can choose between two categories of XDM fields:
+* [!UICONTROL Select attributes]: use this option to map a specific field from your XDM schema to [!DNL Moengage] attribute.
+
+![Moengage Destination Mapping Source Attribute](../../assets/catalog/mobile-engagement/moengage/mapping-attributes.png)
+
+Choose your source field, then click **[!UICONTROL Select]**.
+
+In the [!UICONTROL Target Field] section, click the mapping icon to the right of the field.
+   
+![Moengage Destination Target Mapping](../../assets/catalog/mobile-engagement/moengage/mapping-target.png)
+
+In the [!UICONTROL Select target field] window, you can choose between two categories of target fields:
+* [!UICONTROL Select identity namespace]: Use this option to map [!DNL Platform] identity namespaces to [!DNL Braze] identity namespaces.
```

Comment:
change Braze to Moengage please here and throughout the documentation.

Reply:
done